src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/RecommendationToPartADataMapper.kt

```diff
@@ -16,7 +17,25 @@ class RecommendationToPartADataMapper {
       val recallType = findRecallTypeToDisplay(recommendation.data.recallType)
       val isThisAnEmergencyRecall = convertBooleanToYesNo(recommendation.data.isThisAnEmergencyRecall)
 
-      return PartAData(custodyStatus, recallType, recommendation.data.responseToProbation, isThisAnEmergencyRecall)
+      val selectedAlternativesMap = convertToMap(recommendation.data.alternativesToRecallTried?.selected)
+
+      return PartAData(custodyStatus, recallType, recommendation.data.responseToProbation, isThisAnEmergencyRecall, selectedAlternativesMap)
+    }
+
+    private fun convertToMap(selectedAlternatives: List<SelectedAlternative>?): Map<String, String> {
```

Comment:
Is it worth changing the name of this function to be specific for the SelectedAlternative?

Reply:
Done